DribbbleReader/ImageModalViewController.swift

```diff
@@ -37,44 +37,10 @@ class ImageModalViewController: UIViewController {
     }
     
     func shareAlert(){
-        let actionSheet:UIAlertController = UIAlertController(
-            title:"Share this image",
-            message: self.navigationItem.title,
-            preferredStyle: UIAlertControllerStyle.ActionSheet)
         
-        let cancelAction = UIAlertAction(title: "Cancel", style: .Cancel) { (action) -> Void in
-        }
+        let image = imageView.image
+        let activityVC = UIActivityViewController(activityItems: [image!], applicationActivities: nil)
```

Comment:
Why did you remove twitter and facebook share functions? :dizzy_face: I want to share a image.